### ISSUE TITLE
Privacy plugin doesn't fetch Google fonts properly

### ISSUE THREAD

**Issue by wilhelmer** [squidfunk/mkdocs-material#4514]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

With the changes introduced in insiders-4.26.0, the privacy plugin doesn't properly fetch the Google fonts anymore.

To reproduce, create a minimal MkDocs project using the `mkdocs.yml` below. Build and **serve** the project via `mkdocs serve`. (The bug does **not** occur when the project is opened locally, i.e., via file:// protocol)

The fonts aren't rendered properly, and there's an error message in the console:

```
The stylesheet http://127.0.0.1:8000/assets/external/fonts.googleapis.com/css was not loaded because its MIME type, “application/octet-stream”, is not “text/css”.
```

### Expected behaviour

Fonts should be fetched properly.

### Actual behaviour

See description.

### Steps to reproduce

See description.

### Package versions

- Python: 3.10.8
- MkDocs: 1.4.1
- Material: mkdocs-material-8.5.6-insiders-4.26.0

### Configuration

```yaml
site_name: Test Project
site_url: https://test.com
theme:
    name: material
use_directory_urls: false
nav:
  - Welcome: index.md
plugins:
  - privacy
```

### System information

- Operating system: macOS Ventura
- Browser: Firefox

**Comment by squidfunk** [comment]
Thanks for reporting. Should be fixed in squidfunk/mkdocs-material-insiders@7ab6c9de7. It seems that I mixed up the paths.

**Comment by squidfunk** [comment]
Released as part of 8.5.7+insiders-4.26.1.
